uploader/models.py

```diff
@@ -1,3 +1,23 @@
 from django.db import models
 
-# Create your models here.
+from django.utils import timezone
+
+
+class FileUpload(models.Model):
+
+    number_of_files = models.PositiveIntegerField("Number of Files Uploaded", default=0)
+    username = models.CharField("Username", max_length=80)
+    uploaded_at = models.DateTimeField("Uploaded at", auto_now=True, editable=False)
+
+    def __str__(self):
+        return (
+            self.username
```

Comment:
lets use f strings please.

Reply:
Since the project is on python 3.5.3, I'll use ```str.format()``` .

Reply:
I tried using python 3.6 but few package version are not compatible. Will try to port all packages to python 3.6 too later on